vuu-ui/packages/vuu-shell/src/theme-provider/ThemeProvider.tsx

```diff
@@ -48,27 +62,30 @@ const createThemedChildren = (
 };
 
 interface ThemeProviderProps {
-  children: ReactElement;
+  children?: ReactElement;
   density?: Density;
-  theme?: string;
-  applyClassesToChild?: true;
+  // theme?: Theme;
+  themeMode?: ThemeMode;
+  applyClassesTo?: TargetElement;
 }
 
 export const ThemeProvider = ({
   children,
-  density: densityProp,
-  theme: themeProp,
+  // theme: themeProp,
+  themeMode: themeModeProp,
+  density: densityProp
 }: ThemeProviderProps) => {
```

Comment:
we've lost the code that inherits values from a higher level ThemeProvider. This is very important